target-chains/cosmwasm/contracts/README.md

```diff
@@ -1,3 +1,68 @@
-# Terra Wormhole Contracts
+# Pyth Cosmwasm
 
-The Wormhole Terra integration is developed and maintained by Everstake / @ysavchenko.
+This crate provides utilities to interact with the Pyth contract on the CosmWasm ecosystem.
+It also includes an [example contract](../examples/cw-contract/) demonstrating how to read price feeds from on-chain CosmWasm applications.
+
+## Installation
+
+Add this crate to the dependencies section of your CosmWasm contract's `Cargo.toml` file:
+
+```
+[dependencies]
+pyth-cosmwasm = { git="https://github.com/pyth-network/pyth-crosschain", rev="5d0acc1", features=["library"] }
```

Comment:
we will release it later, right? It's ok to have it on another PR but let's do it sooner rather than later.

Reply:
Yes, will do that in next PR.